### PR TITLE
Lesson2

### PR DESCRIPTION
**Pull request by ilya-frolov**
Hi everyone. This branch includes 2 homework tasks: 
1. Comparison of two numbers; 
2. Verification if four-digit number consists of different digits.

The second task was fully changed from previous Pull request.

Would be great to get response from you according to correctness and quality of my code.

Best regards

### CODE REVIEW

File: HWTask2/src/com/ilyafrolov/lesson2/HomeWorkTask2_IfFiguresDifferent.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                System.out.println("Some numbers are the same");
+            } else if (n3-n4 == 0) {
+                System.out.println("Some numbers are the same");
+            } else {
+                System.out.println("\nIt is correct that all figures in the number are different \n ");
+            }
+
+        System.out.println("The program is completed successfully");
+    }
+
```
Comment (i-velichko) [inline HWTask2/src/com/ilyafrolov/lesson2/HomeWorkTask2_IfFiguresDifferent.java:46]: Formatting please

Comment (ilya-frolov) [reply HWTask2/src/com/ilyafrolov/lesson2/HomeWorkTask2_IfFiguresDifferent.java:46]: > Formatting please

I'm sorry but I don't understand what exactly you mean.

**Review by Artem-Bobr — APPROVED** [review]

**Review by RomanKashenok — APPROVED** [review]

File: HWTask2/src/com/ilyafrolov/lesson2/HomeWorkTask2_IfFiguresDifferent.java (hunk 1 above)
Comment (ilya-frolov) [reply HWTask2/src/com/ilyafrolov/lesson2/HomeWorkTask2_IfFiguresDifferent.java:48]: Thanks, formatted.

File: HWTask2/src/com/ilyafrolov/lesson2/HomeWorkTask2_Comparison.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            System.out.println("The first number is even number");
+        } else {
+            System.out.println("The first number is odd number");
+        }
+
+        if (i2 % 2 == 0) {
+            System.out.println("The second number is even number");
+        } else {
+            System.out.println("The second number is odd number");
+        }
```
[the remark this answers is not in the document]
Comment (ilya-frolov) [reply HWTask2/src/com/ilyafrolov/lesson2/HomeWorkTask2_Comparison.java:31]: Да, просто тренировался.

**Review by mrchernova — APPROVED** [review]
Checked

**Pull request merged**